tests/data/current_time.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WordpressStubs\Tests;
+
+use stdClass;
+
+use function current_time;
+use function PHPStan\Testing\assertType;
+
+// Integer types
+assertType('int', current_time('timestamp'));
+assertType('int', current_time('U'));
+
+// String types
+assertType('string', current_time('mysql'));
+assertType('string', current_time('Hello'));
+
+// Unknown types
+assertType('int|string', current_time($_GET['foo']));
+assertType('int|string', current_time(get_option('date_format')));
+
+// Unsupported types
+assertType('string', current_time(new stdClass()));
+assertType('string', current_time(false));
```

Comment:
beware, those 2 were `int|string` before! I don't have the mental capacity to think this through anymore, better somebody double-check if this is fine

Reply:
I noticed this as well and chose to ignore it because PHPStan will throw an error about `stdClass` and `false` not being a string.

Reply:
ok thx, I thought so but wasn't sured and didn't want to speculate. also, from a type perspective it makes sense I think, it will definitely never be int.

Reply:
https://github.com/WordPress/wordpress-develop/blob/b98b5e59d72a68bc31f99b866fa26f7769d16e97/src/wp-includes/functions.php#L71

Reply:
Agreed that the only time it'll return a string is when `$type` is `'timestamp'` or `'U'`.